tower_cli/models/base.py

```diff
@@ -751,13 +751,31 @@ def lookup_stdout(self, pk=None, start_line=None, end_line=None, full=True):
     @resources.command
     @click.option('--start-line', required=False, type=int, help='Line at which to start printing the standard out.')
     @click.option('--end-line', required=False, type=int, help='Line at which to end printing the standard out.')
-    def stdout(self, pk, start_line=None, end_line=None, **kwargs):
+    def stdout(self, pk, start_line=None, end_line=None, outfile=sys.stdout, **kwargs):
```

Comment:
`outfile` just gets punted on to `click.echo` - passing in a filename rather than a file handle would be an error. 

I could extend this to handle filenames and CLI arguments if that's a blocker to this change.

Reply:
Without the CLI argument, I'm not sure I understand what the use-case is. Will the tower modules consume this option? If so, I would like to get the CLI argument in, because adding it will probably change the call pattern, so might as well finalize that call pattern before someone starts using it.

Reply:
I have a proxy in front of AWX to support a simpler API and it uses the tower-cli module because it abstracts the REST API nicely.

I'll add the CLI option in the next few days.